toolkit/tools/imagecustomizer/main.go

```diff
@@ -18,15 +18,17 @@ import (
 var (
 	app = kingpin.New("imagecustomizer", "Customizes a pre-built CBL-Mariner image")
 
-	buildDir          = app.Flag("build-dir", "Directory to run build out of.").Required().String()
-	imageFile         = app.Flag("image-file", "Path of the base CBL-Mariner image which the customization will be applied to.").Required().String()
-	outputImageFile   = app.Flag("output-image-file", "Path to write the customized image to.").Required().String()
-	outputImageFormat = app.Flag("output-image-format", "Format of output image. Supported: vhd, vhdx, qcow2, raw.").Required().Enum("vhd", "vhdx", "qcow2", "raw")
-	configFile        = app.Flag("config-file", "Path of the image customization config file.").Required().String()
-	logFile           = exe.LogFileFlag(app)
-	logLevel          = exe.LogLevelFlag(app)
-	profFlags         = exe.SetupProfileFlags(app)
-	timestampFile     = app.Flag("timestamp-file", "File that stores timestamps for this program.").String()
+	buildDir                 = app.Flag("build-dir", "Directory to run build out of.").Required().String()
+	imageFile                = app.Flag("image-file", "Path of the base CBL-Mariner image which the customization will be applied to.").Required().String()
+	outputImageFile          = app.Flag("output-image-file", "Path to write the customized image to.").Required().String()
+	outputImageFormat        = app.Flag("output-image-format", "Format of output image. Supported: vhd, vhdx, qcow2, raw.").Required().Enum("vhd", "vhdx", "qcow2", "raw")
+	configFile               = app.Flag("config-file", "Path of the image customization config file.").Required().String()
+	rpmSources               = app.Flag("rpm-source", "Path to a RPMs tarball or a directory containing RPMs.").Strings()
```

Comment:
Our other tools support `--repo-file` to allow a set of custom repos to be passed (via `REPO_LIST="a.repo b.repo c.repo"`). Any interest in doing the same here? (The caveat being they rely on `tdnf` to install them. Maybe we can use the existing image fetcher tool to acquire the desired packages?)

Reply:
This flag can take any of the following:

- An RPM repo config file.
- A directory containing RPMs files.

So, it includes the `--repo-file` functionality. But is also more generic.

The image customizer tool is using `tdnf` to do the installation.

What would be the advantage of using the fetcher instead of just using `tdnf` directly?

Reply:
Major benefit is you don't need to use/modify the real image to deal with networking, but that isn't a huge deal.

Reply:
I figure, giving the custom scripts access to the network in case the users wants/needs it, is probably goodness anyway.